src/util/get-body-size.ts

```diff
@@ -0,0 +1,42 @@
+import type { Readable } from 'node:stream';
+import { HttpBody } from '../types';
+import is from '@sindresorhus/is';
+
+export function getBodySize(
```

Comment:
+1 for using an exported function.  It should be simple to write some true UT, which jsforce lacks, for this.